[docs] Document experimental run attributes (#2141)
vercel/workflow: docs/content/docs/v5/api-reference/workflow/experimental-set-attributes.mdx

---
title: experimental_setAttributes

> description: Attach experimental string metadata to the current workflow run.

Author: ```suggestion
description: Attach string metadata to workflow run for observability.
```

type: reference
summary: Use experimental_setAttributes inside a workflow function to set run attributes.
prerequisites:
  - /docs/foundations/workflows-and-steps
related:
  - /docs/observability/attributes
  - /docs/api-reference/workflow/fatal-error
---

> Attaches plaintext string metadata to the current workflow run.

Author: ```suggestion
Attaches string metadata to the current workflow run.
```
